packages/core/ios/Tests/DdSdkTests.swift

```diff
@@ -784,6 +792,17 @@ internal class DdSdkTests: XCTestCase {
         let mappedEvent = actionEventMapper(mockActionEvent)
         XCTAssertNotNil(mappedEvent)
     }
+
+    func testReactNativeThreadMonitorsRunOnBridge() throws {
+        let bridge = DispatchQueueMock()
+        let mockJSRefreshRateMonitor = MockJSRefreshRateMonitor()
+
+        DdSdkImplementation(mainDispatchQueue: DispatchQueueMock(), bridge: bridge, jsRefreshRateMonitor: mockJSRefreshRateMonitor).initialize(configuration: .mockAny(longTaskThresholdMs: 0.2), resolve: mockResolve, reject: mockReject)
+
+        XCTAssertTrue(bridge.isSameQueue(queue: mockJSRefreshRateMonitor.jsQueue!))
```

Comment:
Might be personal preference here, but in reality we're checking if:
```jsRefreshRateMonitor.startMonitoring()``` was called with correct parameters.

Maybe worth injecting spy JSRefreshRateMonitor and check this instead.